### DISCUSSION TITLE
Allow WebP Images

### DISCUSSION THREAD

**Discussion by joaquinabenza** [gameserverapp/Platform#803] [Suggestions]
Good Morning, I have a advice/suggestion:

Please allow us to upload WebP images on the community/shop pages.

WebP provides smaller file sizes and faster loading times, making it ideal for web use. It also supports transparency and animation, making it a versatile image format.

Furthermore, the latest versions of the most used browsers, such as Chrome and Mozilla, use WebP and do not allow downloading/ Save as... another image format by default. And it's a pain to have to convert every downloaded image to a worse format to use it on your awesome community website.

Thanks!

**Comment by mvaessen** [comment]
going to reject this one, since the images are also used in-game, and games usually only support jpg and png. This is why there are restrictions.
